src/main/java/backend/github/UpdateIssuesTask.java

```diff
@@ -26,16 +27,21 @@ public UpdateIssuesTask(TaskRunner taskRunner, Repo repo, Model model) {
     public void run() {
         ImmutableTriple<List<TurboIssue>, String, Date> changes = repo.getUpdatedIssues(model.getRepoId(),
             model.getUpdateSignature().issuesETag, model.getUpdateSignature().lastCheckTime);
+        List<PullRequest> updatedPullRequests = repo.getUpdatedPullRequests(
+                model.getRepoId(), "", model.getUpdateSignature().lastCheckTime);
```

Comment:
Yup, it's there just to conform to the interface. Since we are implementing our own `since` (which cancel the page downloading early, the ETags don't really help much but can make things confusing to reason about and require us to maintain and store them.